descriptors/ModuleDescriptor-template.json

```diff
@@ -9,34 +9,69 @@
       "handlers": [
         {
           "methods": ["GET"],
-          "pathPattern": "/invoice",
-          "permissionsRequired": ["invoice.collection.get"]
+          "pathPattern": "/invoicing/invoices",
+          "permissionsRequired": ["invoicing.invoice.collection.get"]
         },
         {
           "methods": ["POST"],
-          "pathPattern": "/invoice",
-          "permissionsRequired": ["invoice.item.post"]
+          "pathPattern": "/invoicing/invoices",
```

Comment:
https://dev.folio.org/guidelines/naming-conventions/ . According to the naming conventions
>The first portion is the exact name of the responsible module (back-end modules drop the mod- prefix).

Should we be renaming the module? or the permissions

Reply:
If we don't rename the module then permission names will be like:
`invoice.invoice.item.post` 
`invoice.invoice-lines.item.post` ?

Reply:
I would prefer this:
`invoice.item.post`
`invoice-lines.item.post`
`invoice-number.item.post`
Any thoughts @craigmcnally @piotr-kalashuk ?

Reply:
Damn it, I was unaware of the "**exact** name of the responsible module" part.  

I guess it should be:
`invoice.invoices.item.post` 
`invoice.invoices-lines.item.post`
etc.

Sorry for the confusion @javalkar-varun I should have known about this convention.  Good catch @KVupp

Reply:
No worries. I have corrected permission names.